Library/Formula/avidemux.rb

```diff
@@ -21,29 +29,31 @@ class Avidemux < Formula
   depends_on "pkg-config" => :build
   depends_on "cmake" => :build
   depends_on "yasm" => :build
+  depends_on "xvid" => [:build, :recommended]
+  depends_on "libvpx" => [:build, :recommended]
+  depends_on "libass" => [:build, :recommended]
   depends_on "fontconfig"
   depends_on "gettext"
   depends_on "x264" => :recommended
   depends_on "faac" => :recommended
   depends_on "faad2" => :recommended
   depends_on "lame" => :recommended
-  depends_on "xvid" => :recommended
   depends_on "freetype" => :recommended
-  depends_on "theora" => :recommended
   depends_on "libvorbis" => :recommended
-  depends_on "libvpx" => :recommended
-  depends_on "rtmpdump" => :recommended
   depends_on "opencore-amr" => :recommended
-  depends_on "libvo-aacenc" => :recommended
-  depends_on "libass" => :recommended
-  depends_on "openjpeg" => :recommended
-  depends_on "speex" => :recommended
-  depends_on "schroedinger" => :recommended
-  depends_on "fdk-aac" => :recommended
-  depends_on "opus" => :recommended
-  depends_on "frei0r" => :recommended
-  depends_on "libcaca" => :recommended
+  depends_on "aften" => :recommended
+  depends_on "libdca" => :recommended
   depends_on "qt" => :recommended
+  depends_on "jack" => :optional
+  depends_on "two-lame" => :optional
+  depends_on "fribidi" => :optional
+  depends_on "sdl2" => :optional
+
+  # fix compilation against SDL
+  patch do
+    url "https://gist.githubusercontent.com/Noctem/2c03f24daf6705964347/raw/dd6d27374e5ed44678f25b84da159ea57a9d8857/avidemux-sdl.patch"
+    sha256 "17cb0181fbe503e84ad9b2762eb182ad4ec3ef8df8a90741e4f3c0e4d0a8f1ff"
+  end if build.with? "sdl2"
```

Comment:
Use a block-form `if` (easier to read).

Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!